images/scripts/lib/build-deps.sh

```diff
@@ -25,12 +25,17 @@ esac
 
 echo "$spec" | rpmspec -D "$OS_VER" -D 'version 0' --buildrequires --query /dev/stdin | sed 's/.*/"&"/' | tr '\n' ' '
 
-# We build anaconda rpms on fedora-37
-if [ "$OS_VER" = 'fedora 37' ]; then
-    $GET https://raw.githubusercontent.com/rhinstaller/anaconda/master/anaconda.spec.in | \
-        sed 's/@PACKAGE.*@/0/' | rpmspec --buildrequires --query /dev/stdin | \
-        sed 's/.*/"&"/' | tr '\n' ' '
-fi
+# We build anaconda rpms
+case "$OS_VER" in
+    fedora*) # let's try if Anaconda have branch for the given fedora, if not, use master (rawhide)
+        anaconda_branch="fedora-${OS_VER#fedora }"
```

Comment:
Unused variable. Would actually look a little cleaner if you used it below, but leaving that up to you.

Reply:
Fixed, I'll remove it. Seems to me the same honestly.